### DISCUSSION TITLE
How many cloneid has one cell?

### DISCUSSION THREAD

**Discussion by yueli8** [milaboratory/mixcr#1647] [Q&A] [answered]
Hello,

In my understading, one cell has one cloneid.

In column B, `tagValueCELL` is the barcode of one cell. In row 2 and 3, it is `AAACCTGAGACTCGGA` represented one cell. It has two cloneids: `0` and `2`. `0` is `TRBV10-2`, and `2` is `TRBV5-6`.

I was confuing about that.

Thanks in advance for great help!

Best,

Yue

![捕获](https://github.com/milaboratory/mixcr/assets/40526984/e9125d2b-c912-4682-804e-7eb631e57b68)

**Comment by mizraelson** [comment, marked as answer]
In theory, each cell typically contains one clone. However, contaminations can result in the detection of multiple clones within a sample. Additionally, it is possible for a second chain to be expressed as well.

**Reply by yueli8** [reply]
@mizraelson 

Thank you so much for your quick response!

So the cloneid `2` expressed as well as a real clone, and cloneid `0` may be contaminated?

Is that correct?

Thank you again for your great help!

Best,

Yue

**Reply by mizraelson** [reply]
The accurate way to state it is that both clones, labeled `0` and `2`, were detected in the cell in significant amounts. Had the quantities of either clone been insubstantial, they would have been filtered out during analysis.

**Reply by yueli8** [reply]
@mizraelson Thank you again for your great help!
Best, Yue
